pages/en/lb4/Server.md

```diff
@@ -0,0 +1,220 @@
+---
+lang: en
+title: 'Server'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Server.html
+summary:
+---
+
+Server is a system that enters a working state of doing something by starting it, 
+and exits the working state by stopping it.
+Usually a Server is responsible for interacting with events outside, processing incoming things
+and giving corresponding responses.
+
```

Comment:
This paragraph is too generic. It would be better to describe `Server` in the context of LoopBack 4. We introduce `Server` to represent inbound transports and/or protocols. For example:

- REST over http/https 
- gRPC over http2
- GraphQL over http/https
- jsonrpc over http/https
- MQTT

Reply:
@raymondfeng Part of why there's an explanation regarding Servers as a simpler concept here is that there's nothing to stop a user from creating worker-like implementations that follow this interface. 

As an example, creating processors that convert documents from one format to another in the background do not need to run their own HTTP/S servers (though they'll likely connect to a datasource of some kind in the background).

I asked @jannyHou to cover off this concept since leveraging it gives you two things for free:
- automatic starting/stopping of those instances along with any other Servers
- automatic naming of bound instances for later use

That being said, the general concept of what a Server is shouldn't be in this document.

Reply:
@raymondfeng @kjdelisle @bajtos Shall we agree on the concept of `Server` first?